### ISSUE TITLE
Export snakeCase and camelCase conversion functions

### ISSUE THREAD

**Issue by VladShcherbin** [Vincit/objection.js#781]
It would be great to have them as helper functions for such cases when you need to convert some value (to pass into `where` function for example):

```js
// attribute - external value in camel case
const convertedAttribute = snakeCase(attribute)
const page = await Page.query().where(convertedAttribute, 'some').first()
```

**Comment by koskimas** [comment]
[propertyNameToColumnName](https://github.com/Vincit/objection.js/blob/master/doc/includes/API.md#propertynametocolumnname) function will be available in 1.0 (already is in 0.9.x, but not documented).

**Comment by VladShcherbin** [comment]
@koskimas very cool, thanks 😉
